common/protos/common/asset_locks.proto

```diff
@@ -28,27 +28,28 @@ message AssetLockHTLC {
   HashMechanism hashMechanism = 1;
   bytes hashBase64 = 2;
   uint64 expiryTimeSecs = 3;
-  enum TimeSpec {
+  TimeSpec timeSpec = 4;
+}
+
+enum TimeSpec {
```

Comment:
Given that we've moved `TimeSpec` to the `common` namespace, I wonder if we should name it `LockTimeSpec`, because it is specific to how a lock's time is represented.